src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,32 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet("/driver/add")
```

Comment:
```suggestion
@WebServlet("/drivers/add")
```
it is better to use plural here, and don't forget to change the link in jsp